test/unit-test/FreeRTOS_DNS_Parser/FreeRTOS_DNS_Parser_stubs.c

```diff
@@ -54,6 +54,13 @@ void vPortExitCritical( void )
 
 uint16_t usPacketIdentifier;
 
+
```

Comment:
Please follow the file format in stub.c as well.

Reply:
Updated.